src/matrix_schema/datamodel/pandera.py

```diff
@@ -38,25 +43,32 @@ def get_matrix_node_schema(validate_enumeration_values: bool = True):
     else:
         category_checks = []
 
+    if validate_enumeration_values:
+        all_categories_checks = [pa.Check.isin([category.value for category in NodeCategoryEnum])]
+    else:
+        all_categories_checks = []
+
     return DataFrameSchema(
         columns={
-            "id": Column(T.StringType(), nullable=False),
+            "id": Column(T.StringType(), nullable=True),
```

Comment:
Since you ordered the output now, why are there so many changes to the schema? for example nullable True seems like a big change?

Reply:
I think we need to set required: true on a whole bunch of slots